tools/cross/format.py

```diff
@@ -113,21 +126,61 @@ def git_get_modified_files(
         ).splitlines()
         return files_in_diff
 
+@dataclass
+class FormatConfig:
+    directory: str
+    extensions: Tuple[str, ...]
+    formatter: Callable[[List[str], bool], bool]
+
+
+FORMATTERS = [
+    FormatConfig(
+        directory='src/workerd',
+        extensions=('.c++', '.h'),
+        formatter=clang_format
+    ),
+    FormatConfig(
+        directory='.',
+        extensions=('.js', '.ts', '.cjs', '.ejs', '.mjs'),
+        formatter= prettier
```

Comment:
```suggestion
        formatter=prettier
```

Reply:
We need a python formatter as well 😆

Reply:
Which one do we like? I used `autopep8` at my old job but not sure if there's a cooler one we should be using instead.

Reply:
I think black is currently the best one out there but a lot of people are swearing by ruff because it's written in rust.

Reply:
I found black produced some really ugly output back when I tested it. I  guess I'll go with ruff

Reply:
I think you should probably ask whoever is in charge of most of our python code, is that @hoodmane maybe?

Reply:
Discussion can be continued in #2509